database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,10 +15,9 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
-            $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
+            $table->string('username')->unique();
```

Comment:
De `username` kan toch niet unique zijn want zo kan er een `pietje` in klas 3a zijn en een `pietje` in klas 3b